core/core-configuration-layer.el

```diff
@@ -62,6 +62,10 @@
   (expand-file-name (concat user-emacs-directory "private/"))
   "Spacemacs private layers base directory.")
 
+(defconst configuration-certs-directory
```

Comment:
I was missing the `layer` part when copying preceding `defconst` lines, shame on me :sweat: . Why should this one be private and not the others? I mean, it's fine for me, just wan to know :relaxed: